packages/tests/.mocharc.cjs

```diff
@@ -7,7 +7,7 @@ const config = {
     'loader=ts-node/esm'
   ],
   exit: true,
-  retries: 4
```

Comment:
shouldn't we keep it?

Reply:
correct. some debugging-related change slipped in. reverted